page-info.php

```diff
@@ -19,7 +19,7 @@
 ?>
       <section class="grid-row">
         <div class="grid-item item-m-4">
-          <h2>About</h2>
+          <h2 class="font-heading font-uppercase font-size-heading">About</h2>
```

Comment:
Its kinda annoying that `.font-heading` is so close to the size class. I wonder if it should be `.font-family-heading` or `.font-typeface-heading` or `.font-file-heading` ?